src/components/Layout/Layout.tsx

```diff
@@ -126,12 +130,8 @@ function ContentArea() {
         justifyContent="end"
         top={(theme) => theme.spacing(1)}
       >
-        {!isViewingPresentation && (
-          <>
-            <BoardBar />
-            <CollaborationBar />
-          </>
-        )}
+        {!isViewingPresentation && <BoardBar />}
+        {isCollaborationBarActive && <CollaborationBar />}
```

Comment:
I would directly put 2 conditions  (is edit is enabled or `!isViewingPresentation`) here to see explicitly when `CollaborationBar` is rendered.